### PR TITLE
CONCF-768 add optimizely events for featured content

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/CONCF-768

### CODE REVIEW

File: front/scripts/main/components/FeaturedContentComponent.ts [hunk 1]
```diff
@@ -22,16 +22,19 @@ App.FeaturedContentComponent = Em.Component.extend(App.FeaturedContentMixin, App
 
 	rightClickHandler: function(): boolean {
 		this.nextItem();
+		Mercury.Utils.VariantTesting.trackEvent('featured-content-next');
```
Comment (rogatty) [inline front/scripts/main/components/FeaturedContentComponent.ts]: You can use `M.VariantTesting` instead.

File: front/scripts/main/components/FeaturedContentVariation1Component.ts [hunk 2]
```diff
@@ -10,15 +10,18 @@ App.FeaturedContentVariation1Component = Em.Component.extend(App.FeaturedContent
...
 
 		swipeRight: function (): void {
+			Mercury.Utils.VariantTesting.trackEvent('featured-content-prev');
 			this.prevItem();
 		},
 	},
 
 	click: function (): void {
+		Mercury.Utils.VariantTesting.trackEvent('featured-content-click');
 		this.trackClick('modular-main-page', 'featured-content');
```
Comment (rogatty) [inline front/scripts/main/components/FeaturedContentVariation1Component.ts]: Shouldn't we remove this one?

**Comment by rogatty** [comment]
LGTM

File: front/scripts/main/components/FeaturedContentComponent.ts [hunk 3]
```diff
@@ -22,20 +22,23 @@ App.FeaturedContentComponent = Em.Component.extend(App.FeaturedContentMixin, App
 
 	rightClickHandler: function(): boolean {
 		this.nextItem();
+		M.VariantTesting.trackEvent('featured-content-next');
```
Comment (Warkot) [inline front/scripts/main/components/FeaturedContentComponent.ts]: Probably it won't make much difference, but shouldn't we implement it the same way as it it done for the rest of the variants and their actions — first track, then run `next`/`prevItem()`?

**Comment by Warkot** [comment]
LGTM

**Pull request merged**